src/main/java/net/minecraftforge/common/ForgeHooks.java

```diff
@@ -958,25 +958,31 @@ public static Vec3d rayTraceEyeHitVec(EntityLivingBase entity, double length)
         return git == null ? null : git.hitVec;
     }
 
-    public static boolean onInteractEntityAt(EntityPlayer player, Entity entity, RayTraceResult ray, ItemStack stack, EnumHand hand)
```

Comment:
I don't understand how else I'm supposed to pass an EnumActionResult back to vanilla if I can't change these